sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/PostProcessor.java

```diff
@@ -8,8 +8,8 @@
 import com.azure.storage.common.Utility;
 import reactor.core.publisher.Mono;
 
-final class PostProcessor {
-    static <T> Mono<T> postProcessResponse(Mono<T> response) {
+public final class PostProcessor {
+    public static <T> Mono<T> postProcessResponse(Mono<T> response) {
```

Comment:
Does this class need to become public? It seems unnecessary from an end user perspective?

Reply:
Moved this into the implementation package